tmva/sofie/src/RModel.cxx

```diff
@@ -797,7 +804,7 @@ long RModel::WriteInitializedTensorsToFile(std::string filename) {
 void RModel::PrintRequiredInputTensors() {
     std::cout << "Model requires following inputs:\n";
     for (auto& inputInfo: fInputTensorInfos) {
-        std::cout << "Parameterised Tensor name: " << inputInfo.first << "\t";
+        std::cout << "Parametraised Tensor name: " << inputInfo.first << "\t";
```

Comment:
I guess it was in the correct spelling previously.

Reply:
Correct! Thanks